### DISCUSSION TITLE
How to set a custom domain during build and deployment?

### DISCUSSION THREAD

**Discussion by mihaiconstantin** [crazy-max/ghaction-github-pages#202] [Q&A] [answered]
Thanks for this handy action!

I wonder whether it's possible to run the action with a custom domain.
At the moment, after each deployment, I need to go to the settings to add the custom domain, which will result in a new commit on the `gh-pages` branch that will add the `CNAME` file.

I see in the documentation the `domain` input, but I am not sure if this is indeed the one I am looking for.

| Name     | Type   | Description                       |
| -------- | ------ | --------------------------------- |
| `domain` | String | Git domain (default `github.com`) |

**Comment by aklinker1** [comment, marked as answer]
I had the same question today, use the `fqdn` option, not sure what it stands for, but it worked perfectly!

![image](https://user-images.githubusercontent.com/10101283/183321338-31db104c-d9a3-4f59-894a-1421b0051559.png)

**Comment by mihaiconstantin** [comment]
> I had the same question today, use the `fqdn` option, not sure what it stands for, but it worked perfectly!
> 
> ![image](https://user-images.githubusercontent.com/10101283/183321338-31db104c-d9a3-4f59-894a-1421b0051559.png)

Thanks for the tip! I suppose we can close this?
